docs/source/markdown/podman-build.1.md

```diff
@@ -111,17 +111,19 @@ network namespaces can be found.
 
 **--cpu-period**=*limit*
 
-Limit the CPU CFS (Completely Fair Scheduler) period
-
-Limit the container's CPU usage. This flag tell the kernel to restrict the container's CPU usage to the period you specify.
+Limit the container's CPU usage by setting CPU CFS (Completely Fair Scheduler)
+period. The CPU period is a duration in microseconds. Once the container's CPU
+quota is used up, it will not be scheduled to run until the current period
+ends. Defaults to 100000 microseconds.
```

Comment:
Can you it a bit more explicit that input is interpreted as a number of microseconds?

Reply:
Sure. How about, "--cpu-period: Set the CPU period for the Completely Fair Scheduler (CFS), which is a duration in microseconds."

Reply:
Sure, SGTM